src/test/java/io/confluent/examples/streams/ApplicationResetIntegrationTest.java

```diff
@@ -133,15 +138,19 @@ public void shouldReprocess() throws Exception {
     //
     // Step 5: Rerun application
     //
-    streams = ApplicationResetExample.run(true, streamsConfiguration);
+    final KafkaStreams streams1 = ApplicationResetExample.buildKafkaStreams(streamsConfiguration);
```

Comment:
`streams1` is a very bad name -- also, logically, we restart the same application, thus reusing existing variable seems sound to me.

Reply:
Ok.

Reply:
I've refactored it using code blocks so that we can use the same name without re-assigning the same variable.

Hopefully, this is the best of both worlds.